packages/hydrogen-react/src/CartProvider.stories.tsx

```diff
@@ -246,7 +246,7 @@ Default.args = {
   numCartLines: 30,
   /** A callback that is invoked when the process to create a cart begins, but before the cart is created in the Storefront API. */
   data: undefined,
-  /** A fragment used to query the Storefront API's [Cart object](https://shopify.dev/api/storefront/2023-10/objects/cart) for all queries and mutations. A default value is used if no argument is provided. */
+  /** A fragment used to query the Storefront API's [Cart object](https://shopify.dev/api/storefront/2024-01/objects/cart) for all queries and mutations. A default value is used if no argument is provided. */
```

Comment:
Just curious, whats the difference b/t current and latest? I was told that we can't use latest since old doc may stop working but current will fix it?

cc @blittle

Reply:
Correct. We should _not_ use `current`, because it will break on older versions of the docs.